### ISSUE TITLE
Internal Server Error 500

### ISSUE THREAD

**Issue by Dominik1999** [raiden-network/raiden-installer#79]
In the case, where a user downloads a new Wizard version but has some old config files, the Wizard returns an `Internal Server Error 500`.

With ubuntu, we need to delete all görli related files in `.local/share/Raiden/keystore`.

We need to handle that error better.

**Comment by czepluch** [comment]
I assume that by "handle that error better" you mean that we don't delete any keystore files, but instead rename the dir or something like that?

**Comment by Dominik1999** [comment]
Yes, I do not know the solution tbh. We just need to make sure that the user doesn't see the "Internal Server Error 500" This is not user friendly
